test/units/testsuite-49.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+set -ex
+set -o pipefail
+
+modprobe loop
+
+cd /tmp
+
+mkdir -p img/usr/lib img/etc
```

Comment:
we nowadays run most tests in the same image, hence each test should either use random test dirs or uniquely named ones, so that they are unlikely to conflict with other tests that might run in parallel. i.e. "img" is too generic a subdir, might conflict with another test.

Reply:
Isn't /tmp on a tmpfs? But anyway it's a good point, I'll create a random dir with mktemp, it seems good practice